### ISSUE TITLE
"ionic build ios" failed

### ISSUE THREAD

**Issue by kcampion** [nordnet/cordova-hot-code-push-local-dev-addon#2]
When I use `ionic build ios` this mistake appears:

```
Error: Cannot find module 'colors/safe'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at /Users/me/Documents/my_app/plugins/cordova-hot-code-push-local-dev-addon/scripts/lib/logger.js:7:16
    at Object.<anonymous> (/Users/me/Documents/my_app/plugins/cordova-hot-code-push-local-dev-addon/scripts/lib/logger.js:93:3)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
```

**Comment by kcampion** [comment]
To solve this problem, edit .../plugins/cordova-hot-code-push-local-dev-addon/scripts/lib/logger.js at line 7 and replace this line:

```
var colors = require('colors/safe'),
```

by

```
var colors = require('colors'),
```

**Comment by nikDemyankov** [comment]
Can't reproduce that issue, but will do the change in `require`. The only idea is that `colors` module was already installed on your computer, and it is an older version than the one that is used in the plugin.

**Comment by nikDemyankov** [comment]
Fixed as you suggested. Released v0.1.1
